weave/basket.py

```diff
@@ -259,14 +276,147 @@ def ls(self, relative_path=None):
         # not passing refresh=True
         return self.file_system.ls(ls_path, refresh=True)
 
+    # pylint: disable-msg=duplicate-code
     def to_pandas_df(self):
         """Return a dataframe of the basket member variables."""
-        data = [self.uuid, self.upload_time,
-                self.parent_uuids, self.basket_type,
-                self.label, self.weave_version, self.address,
-                self.storage_type]
-        columns = ["uuid", "upload_time", "parent_uuids",
-                   "basket_type", "label", "weave_version",
-                   "address", "storage_type"]
+        data = [
+            self.uuid,
+            self.upload_time,
+            self.parent_uuids,
+            self.basket_type,
+            self.label,
+            self.weave_version,
+            self.address,
+            self.storage_type,
+        ]
+        columns = [
+            "uuid",
+            "upload_time",
+            "parent_uuids",
+            "basket_type",
+            "label",
+            "weave_version",
+            "address",
+            "storage_type",
+        ]
 
         return pd.DataFrame(data=[data], columns=columns)
+
+# pylint: disable-msg=too-many-locals
+def create_basket_in_place(directory_path, **kwargs):
+    """Creates a basket in place by generating the manifest, supplement, and
+    metadata files directly in the provided directory without moving or
+    uploading any files.
```

Comment:
As per pep-8, the first line of the docstring should be a single-line summary, followed by a blank line, followed by a more elaborate description